include/git2/indexer.h

```diff
@@ -132,6 +136,8 @@ GIT_EXTERN(int) git_indexer_commit(git_indexer *idx, git_indexer_progress *stats
  * names. This is only correct after the index has been finalized.
  *
  * @param idx the indexer instance
+ * @return the packfile's hash. In casewhere the index has not been finalized,
+ *	it will be zeroed out.
```

Comment:
We state that this should only be called when it's been finalized, let's not make promises about what happens if you don't obey that.

```suggestion
 * @return the packfile's hash
```